apps/src/gamelab/levels.js

```diff
@@ -79,6 +79,7 @@ levels.custom = utils.extend(levels.sandbox, {
 
     // Sprites
     "var sprite = createSprite": null,
+    "createEdgeSprites": null,
```

Comment:
Apparently we don't. Ideally they want if someone uses `createEdgeSprites`, then touching methods would look like this:
<img width="319" alt="screen shot 2016-08-04 at 1 34 07 pm" src="https://cloud.githubusercontent.com/assets/4640747/17417425/2900ab14-5a48-11e6-912e-96362da34c6a.png">
I think the closest thing we've done before is the `setAnimation` dropdowns.